spylib/webhook.py

```diff
@@ -18,3 +23,104 @@ class WebhookCreate(Enum):
     HTTP = 'webhookSubscriptionCreate'
     EVENT_BRIDGE = 'eventBridgeWebhookSubscriptionCreate'
     PUB_SUB = 'pubSubWebhookSubscriptionCreate'
+
+
+def is_valid(data: str, hmac_header: str, api_secret_key: str) -> bool:
```

Comment:
Let's have function names that are verbs, and more precisely actions because a function does something on the output to return a results as opposed to being something.

```suggestion
def validate(data: str, hmac_header: str, api_secret_key: str) -> bool:
```

It'll be nice to have `webhook.validate`.

On the other hand using `is_valid` for the name of the boolean from the function is great because that describes the state of the webhook after the validation.